bb8/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bb8"
-version = "0.8.3"
+version = "0.9.0"
```

Comment:
Why not 0.8.4? I think all of your changes have been semver-compatible?

Reply:
Since its adding a new feature and not patching anything got the feeling that minor bump would be the most suitable one to bump.

Reply:
And we keep compatibility

Reply:
That's not how semver works in Cargo: 0.x to 0.y is a semver-incompatible bump, there is no distinction between feature and fix versions in 0.x versions.

Reply:
I was not aware about this change from Cargo compared to what we have historically used major, minor and patched versions.

Reply:
Ill put the 0.8.4

Reply:
Done, and thanks for the insights here about how Cargo works with versions